src/pages/about.js

```diff
@@ -1,114 +1,135 @@
 import { Flex, Heading, useTheme } from '@chakra-ui/core';
+import { graphql, StaticQuery } from 'gatsby';
 import React from 'react';
 import ContactCard from '../components/about/ContactCard';
 import Content from '../components/about/Content';
+import ErrorBoundary from '../components/ErrorBoundary';
 import Image from '../components/Image';
 import Layout from '../components/Layout';
 import { MISSION_MESSAGE, WHO_WE_ARE_MESSAGE } from '../constants/about';
 
 export default function About() {
   const theme = useTheme();
   return (
-    <Layout>
-      <Flex align="center" justify="center" direction="column">
-        <Flex
-          w="100%"
-          minH="260px"
-          align="center"
-          justify="center"
-          zIndex="-1"
-          backgroundColor="#000"
-          color="#FFF"
-        >
-          <Image
-            publicId="assets/people-protesting-on-street-4552840_gginry"
-            cloudName="rebuild-black-business"
-            pos="absolute"
-            objectFit="cover"
-            w="100%"
-            h="100%"
-            zIndex="-1"
-            transforms={{
-              gravity: 'auto',
-              opacity: '50',
-            }}
-          />
-          <Heading fontFamily={theme.fonts['heading-slab']} size="xl">
-            ABOUT
-          </Heading>
-        </Flex>
-        <Flex w="100%" backgroundColor={theme.colors['rbb-white']}>
-          <Flex
-            direction="column"
-            align="center"
-            justify="center"
-            alignItems="flex-start"
-            marginTop={[theme.spacing.base, theme.spacing.base, 0, 0]}
-            marginBottom={['1.125rem', '1.125rem', 0, 0]}
-            marginLeft={[0, 0, '5%', '15%', '15%', '30%']}
-            marginRight={[0, 0, '5%', '20%', '22%', '35%']}
-          >
-            <Content
-              heading="MISSION"
-              message={MISSION_MESSAGE}
-              marginTop="1.9375rem"
-              marginBottom="0"
-              dividerMargin="6.313rem"
-            />
-            <Content
-              heading="WHO WE ARE"
-              message={WHO_WE_ARE_MESSAGE}
-              marginTop="3rem"
-              marginBottom="1.9375rem"
-              dividerMargin="3.9375rem"
-            />
-          </Flex>
-        </Flex>
+    <ErrorBoundary>
+      <StaticQuery
```

Comment:
Wondering if we can maybe wrap this static query around only the part of the markup that is leveraging the queried data?

My lack of Gatsby knowledge here is showing a little bit, but would feel nice from a markup perspective :D

Reply:
I will take a look, what you're saying makes sense

Reply:
should be fixed now.